src/main/resources/assets/admin/common/js/form/FormItemOccurrences.ts

```diff
@@ -157,7 +157,7 @@ export class FormItemOccurrences<V extends FormItemOccurrenceView> {
         });
     }
 
-    createNewOccurrenceView(_occurrence: FormItemOccurrence<V>): V {
+    createNewOccurrenceView(_occurrence: FormItemOccurrence<V>, isNew: boolean = false): V {
```

Comment:
`isNew` argument for `createNewOccurrenceView` doesn't make sense at all... Doesn't it already imply that we are creating a NEW occurrence view? Either the argument has the wrong name, or the method.